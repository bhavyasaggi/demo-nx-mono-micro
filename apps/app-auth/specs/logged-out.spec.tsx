import React from 'react';
import { render } from '@testing-library/react';

import PageLoggedOut from '../pages/sign-in';

describe('PageLoggedOut', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<PageLoggedOut />);
    expect(baseElement).toBeTruthy();
  });
});
