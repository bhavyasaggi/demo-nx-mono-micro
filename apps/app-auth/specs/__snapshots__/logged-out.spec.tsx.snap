// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoggedOut should render successfully 1`] = `
<div>
  <div
    class="css-1lnrks8"
  >
    <div
      class="chakra-card css-2dlz7"
    >
      <div
        class="chakra-card__body css-0"
      >
        <h2
          class="chakra-heading css-0"
        >
          Logged out successfully
        </h2>
        <hr
          aria-orientation="horizontal"
          class="chakra-divider css-1upb9tn"
        />
        <a
          class="chakra-link css-0"
          href="/sign-in"
        >
          Sign In
        </a>
      </div>
    </div>
  </div>
</div>
`;
